tests/integration/helpers/zookeeper_secure_config.xml

```diff
@@ -1,5 +1,5 @@
 <clickhouse>
-    <zookeeper>
+    <keeper>
```

Comment:
test with `keeper_server` is needed.

and don't just change here, create a test which will test 3 different configs each using a different way of defining ZooKeeper cluster.
You create a ReplicatedMergeTree and test that an insert works.